composeApp/src/ooniMain/kotlin/org/ooni/probe/ui/onboarding/OnboardingUtils.kt

```diff
@@ -0,0 +1,54 @@
+package org.ooni.probe.ui.onboarding
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import ooniprobe.composeapp.generated.resources.Common_Clear
+import ooniprobe.composeapp.generated.resources.Res
+import org.ooni.probe.config.OrganizationConfig
+
+import ooniprobe.composeapp.generated.resources.Onboarding_CleanUp_Paragraph
+import ooniprobe.composeapp.generated.resources.Onboarding_CleanUp_Title
+import ooniprobe.composeapp.generated.resources.Onboarding_Crash_Button_No
+
+@Composable
+fun ColumnScope.CleanUpStep(onEvent: (OnboardingViewModel.Event) -> Unit) {
```

Comment:
This could have been implemented for both platforms, I don't see harm in it. That would lower the amount of code for this feature, and keep more functions private.

Reply:
DW has no previous versions per say

Reply:
I know, that's why I say it doesn't matter much. But I guess the onboarding image is OONI specific, so let's leave it.